ProjectLighthouse/ProjectLighthouse/Migrations/DatabaseModelSnapshot.cs

```diff
@@ -406,6 +406,13 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<ulong>("LocationPacked")
                         .HasColumnType("bigint unsigned");
 
+                    b.Property<bool>("LockedByModerator")
+                        .HasColumnType("tinyint(1)");
+
+                    b.Property<string>("LockedReason")
+                        .IsRequired()
```

Comment:
LockedReason is required but has no default value for previous records. This seems like it might cause issues, have you already tested this migration?

Reply:
I had no issues when migrating a copy of the live Beacon database.